### PR TITLE
Welcome page

### PR DESCRIPTION
**Pull request by Jessi2498**

### CODE REVIEW

File: src/App.css
Comment (Rahnken) [inline src/App.css:1]: could have a prettier pass. This will get a bunch of updates, based on my prettier setup.

File: src/App.tsx [hunk 1]
```diff
@@ -1,7 +1,30 @@
 function App() {
   return (
     <>
-      <div> Hello Team !!!</div>
+      <div className="container-nav">
+        <nav>
```
Comment (Rahnken) [inline src/App.tsx:6]: This will end up being replaced when we set up a navbar component , we can discuss why these change in a second.

File: src/App.tsx [hunk 2]
```diff
@@ -1,7 +1,30 @@
 function App() {
```
Comment (Rahnken) [inline src/App.tsx:2]: The reason the css didnt show up is we need to import the css file.

**Review by Rahnken — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 2 above)
Comment (Jessi2498) [reply src/App.tsx:2]: hey i added the import its working now thank you

**Review by Rahnken — APPROVED** [review]
